### ISSUE TITLE
Error with build on Angular since version 12

### ISSUE THREAD

**Issue by edthenet** [cojamru/signalr-no-jquery#5]
When is use the example code:

`import { hubConnection } from '@cojam/signalr-no-jquery';`

I get the following error upon build:

`Error: export 'hubConnection' (imported as 'hubConnection') was not found in '@cojam/signalr-no-jquery' (module has no exports)`

**Comment by eikaramba** [comment]
i have the same problem
```
exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and 'xxxxxxx\node_modules\.pnpm\@cojam+signalr-no-jquery@0.3.0\node_modules\@cojam\signalr-no-jquery\package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```

**Comment by edthenet** [comment]
> i have the same problem
> 
> ```
> exports is not defined in ES module scope
> This file is being treated as an ES module because it has a '.js' file extension and 'xxxxxxx\node_modules\.pnpm\@cojam+signalr-no-jquery@0.3.0\node_modules\@cojam\signalr-no-jquery\package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
> ```

@eikaramba 
I think you solved this issue. Thank you. Do we have to wait for an update or did you make a fork?

I have changed the signalR.js file to signalR.cjs and also changed the reference in the package.json
After that my project runs again, but it still isn't working bacuase i got a error message in the console
`Object.defineProperty(exports, "__esModule", { value: true });`
The i removed the following rule in the package.json
`"type": "module"`
And everthing seems to work now

**Comment by eikaramba** [comment]
i cannot recollect completely whether i was able to solve it here or used another library. there are a few forks from the original one. i just know that i unforunately needed an earlier version of the library and thus ended up just doing it myself. meaning copying the original signalR code and replacing all jquery instances and mocking it. i just followed the commits from this repository. took me only 10 minutes or so and it was working. also i made sure to actually only use es6 exports.

update: it seems i was using @10pearls/signalr-no-jquery inbetween as the latest working non-custom version

**Comment by e965** [comment]
Released a `0.4.0` version where this issue (probably) should be solved. Check, please.

```sh
npm install @cojam/signalr-no-jquery@0.4.0 -D
```

**Comment by e965** [comment]
Closed due to inactivity.
